macantine/etl.py

```diff
@@ -344,8 +345,20 @@ def len_dataset(self):
         else:
             return 0
 
-    def is_valid(self) -> bool:
-        dataset_to_validate_url = f"{os.environ['CELLAR_HOST']}/{os.environ['CELLAR_BUCKET_NAME']}/media/open_data/{self.dataset_name}_to_validate.csv"
+    def is_valid(self, filepath) -> bool:
+        # Saving file online
```

Comment:
pourquoi est-ce que tu fais ça ici ? Il me paraît hors du scope de cette méthode selon le nom `is_valid`

Reply:
La question porte sur le fait d'enregistrer le fichier ? 
Si oui, on doit l'enregistrer en ligne pour pouvoir utiliser l'API de validation

Reply:
ah, j'avais pas compris ça. Alors je propose:
```suggestion
        # In order to validate the dataset with the validata api, must first convert to CSV then save online
```